package.json

```diff
@@ -187,7 +181,7 @@
     "typescript": "5.6.2",
     "typescript-eslint": "^8.3.0",
     "wait-port": "^0.3.0",
-    "nx-cloud": "latest"
+    "nx-cloud": "19.1.0"
```

Comment:
Is it intended for `nx-cloud` to be 1 major version behind?

Reply:
Yes, they haven't released a 20.X yet. PTAL https://www.npmjs.com/package/nx-cloud